### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by manojncsoni007**
Screenshot (172)

### CODE REVIEW

**Comment by manojncsoni007** [comment]
Screenshot (172)

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: Components/cards/card.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../cards/card.css">
+    <script src="https://kit.fontawesome.com/e24bb32a8e.js" crossorigin="anonymous"></script>
```
Comment (arpit01923) [inline Components/cards/card.html:10]: instead of using script you should add link tag

**Review by arpit01923 — APPROVED** [review]
great work Manoj , resolve one minor change

**Pull request merged**